scrapy/settings/__init__.py

```diff
@@ -238,7 +239,7 @@ def getdict(
     def getdictorlist(
         self,
         name: _SettingsKeyT,
-        default: Union[Dict[Any, Any], List[Any], None] = None,
+        default: Union[Dict[Any, Any], List[Any], Set[Any], Tuple[Any], None] = None,
```

Comment:
I was in doubt about add this typing hint, since is for the default value, make sense or should i remove it?

Reply:
I think it makes sense as it's supported, but doesn't matter that much.